net/lnet/st/ping.c

```diff
@@ -1498,7 +1498,7 @@ void nlx_ping_server_spawn(struct m0_thread *server_thread,
 {
 	int rc;
 
-	sctx->pc_xprt = &m0_net_lnet_xprt;
+	sctx->pc_xprt = m0_net_xprt_default_get();
```

Comment:
Here, the same. And all others.
This is the UT or ST code. 
It must use "lnet" to do the test, not the *default* one, because the default one might be some other xprt.